runtime/i18n/host-locale-provider.ts

```diff
@@ -7,7 +7,7 @@ import { Observable } from 'rxjs/Observable';
 
 import {
   SkyAppWindowRef
-} from '@blackbaud/skyux-builder/runtime';
+} from '@blackbaud/skyux-builder/runtime/window-ref';
```

Comment:
This was causing a strange circular reference error (probably because it's pointing to a barrel in its parent directory).